Probesys/bileto
# dev: Refactor the forms

## Related issue(s)


https://github.com/Probesys/bileto/issues/551

## How to test manually



1. Use the forms
2. Verify everything works

## Checklist
